app/frontend/actions/reloadWallet.ts

```diff
@@ -24,6 +24,7 @@ export default (store: Store) => {
   const reloadWalletInfo = async (state: State): Promise<void> => {
     setWalletOperationStatusType(state, 'reloading')
     const wallet = getWallet()
+    wallet.invalidateCache()
```

Comment:
I'd clarify here with a comment why we need to invalidate and what edge-cases are we protecting against